lib/manager/components/validation/ValidationErrorItem.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+
+import Icon from '@conveyal/woonerf/components/icon'
+import moment from 'moment'
+import React, {Component} from 'react'
+import {Button} from 'react-bootstrap'
+import {browserHistory} from 'react-router'
+
+import {isEditingDisabled} from '../../util'
+
+import type {Feed, FeedVersion, Project, ValidationError} from '../../../types'
+import type {GtfsState} from '../../../gtfs/reducers/index'
+import type {UserState} from '../../reducers/user'
+
+type Props = {
+  error: ValidationError,
+  fetchGTFSEntities: any => void,
+  feedSource: Feed,
+  index: number,
+  multipleEntityTypes: boolean,
+  project: Project,
+  gtfs: GtfsState,
+  user: UserState,
+  version: FeedVersion
+}
+
+type State = {
+  expanded: boolean
+}
+
+export default class ValidationErrorItem extends Component<Props, State> {
+  state = {expanded: false}
+
+  _onClickFetch = () => {
+    const {expanded} = this.state
+    const {error, gtfs, version: feedVersion, fetchGTFSEntities} = this.props
+    const {entity_id: id, entity_type: type} = error
+    const entity = gtfs.validation.data.find(e => e._id === `${type}:${id}`)
+    if (!expanded) {
+      if (!entity) {
+        // fetch GTFS entity
+        const {entity_id: id, entity_type: type} = error
+        if (id) {
+          fetchGTFSEntities({type, id: [id], namespace: feedVersion.namespace})
+        } else {
+          // FIXME: fetch by line number
+          fetchGTFSEntities({type, id: [id], namespace: feedVersion.namespace})
+        }
+      }
+    }
+    this.setState({expanded: !expanded})
+  }
+
+  _renderEntityDetails = (entity: any, type: string, sequence: ?number) => {
+    if (entity) {
+      if (type === 'StopTime') {
+        const items = []
```

Comment:
Let's make this into its own StopTimeRow component

Reply:
Since we're not using react 16 (i.e., no support for fragments), I think that would require encapsulating all of the table rows in a `<div>` or some other html element. I'm not sure adding that element is worth breaking out into a separate component.

Reply:
All right, I'm not too opinionated about this.  This could potentially also even happen as inline jsx in the render function.